src/charm.py

```diff
@@ -196,8 +196,14 @@ def _prometheus_configurations(self):
             configured_jobs.append(job)
 
         alerts = list(self._metrics_providers.alerts().values())
-
-        return {"scrape_jobs": configured_jobs, "alert_rules": alerts[0] if alerts else []}
+        alert_groups = {"groups": []}  # type: ignore
+        for entry in alerts:
+            alert_groups["groups"] = alert_groups["groups"] + entry["groups"]
```

Comment:
```suggestion
            alert_groups["groups"] += entry["groups"]
```

Reply:
It will get inlined anyway, so this is preference and readability.

We could also vote for:

```python
alert_groups["groups"].extend(entry["groups"])

# Or...
alert_groups = {
  "groups": [entry["groups"] for entry in alerts]
```

I like that last one, but it doesn't matter that much.